src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const step of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```

Reply:
or we can decompose it 
```suggestion
  for (const { type, extraData, ... } of actions) {
```